src/cli/plugin/plugin_lister.js

```diff
@@ -0,0 +1,12 @@
+const fs = require('fs');
+
+export function list(settings, logger) {
+  const files = fs.readdirSync(settings.pluginDir);
+  files
+  .filter(function (file) {
+    return file[0] !== '.';
```

Comment:
Since you only need the first value of the file array, I'd recommend destructuring the array argument so this conditional becomes a bit more expressive:

```js
.filter(function([ fileName ]) {
  return fileName !== '.';
});
```